app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headlamp",
-  "version": "0.29.0",
+  "version": "0.30.0",
```

Comment:
This is just to make it easier to test installing the minikube plugin, because it requires version 0.30.0.

Reply:
This commit has been removed now.